www/lobby/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Router, Request, Response } from 'express';
+import { Lobby } from '../../lib/lobby';
+import { User } from '../../lib/user';
+import { validateJwt } from '../../utils/jwt';
+import { lobby_id_router } from './:id';
+
+export const lobby_router = Router();
+
+lobby_router.use('/middleware', validateJwt);
+
+// Creates a new lobby and returns the lobby id w/ lobby data
+/**
+ * Body Params: lobbyName, theme, id, refreshToken
+ */
+lobby_router.post('/', async (req: Request, res: Response) => {
+  const lobbyName = req.body.lobbyName;
+  const theme = req.body.theme;
+  const userId = req.body.id;
+  const participants = [userId];
+  const songIds: string[] = [];
```

Comment:
creating these almost seems useless as it is something we can do within the `Lobby.create` function